test/chain.ts

```diff
@@ -21,12 +21,17 @@ describe('chain codes', function() {
     Codes.p2wsh,
   ];
 
-  const unspentTypeList = [
+  const validUnspentTypeList = [
     UnspentType.p2sh,
     UnspentType.p2shP2wsh,
     UnspentType.p2wsh,
   ];
 
+  const invalidUnspentTypeList = [
```

Comment:
They're not "invalid", but rather "unsupported"?

Reply:
Renamed